pengwin-setup.d/common.sh

```diff
@@ -205,29 +205,17 @@ function setup_env() {
 
 function install_packages() {
 
-  if [[ ${NON_INTERACTIVE} ]]; then
-    sudo apt-get install -y -q "$@"
-  else
-    sudo debconf-apt-progress -- apt-get install -y "$@"
-  fi
+  sudo apt-get install -y -q "$@"
 }
 
 function update_packages() {
 
-  if [[ ${NON_INTERACTIVE} ]]; then
```

Comment:
Revert this change. In update there is no problem with the GUI

Reply:
I just make the rollback to the changes inside update_packages method